h2o-r/tests/runitUtils/matchers.R

```diff
@@ -0,0 +1,10 @@
+library(rlang)
```

Comment:
You can remove this, since we will just use `expect_equal()`.  See below.

Reply:
Why was this needed?

Reply:
If we don't want to introduce new methods for clarity than yes.... we should delete this. @navdeep-G expect_defined is just a modified copy of expect_null. This library was required by enquo method.

Reply:
Got it. I think we can forgo adding this now, but something to consider for the future. Not sure if this introduced a new dependency from the R dev side. I am inclined to think it does.